models/DataObject/ClassDefinition/CustomLayout/Dao.php

```diff
@@ -107,7 +107,7 @@ public function getIdByNameAndClassId($name, $classId)
     }
 
     /**
-     * @return int|mixed
+     * @return int
```

Comment:
Should be `int|null` right?

Reply:
Why? If the DB return null, the value will be set to 1

Reply:
You're right, sorry!